### DISCUSSION TITLE
Missing HttpCookie in System.Web

### DISCUSSION THREAD

**Discussion by enfJoao** [oqtane/oqtane.framework#4706] [Q&A] [answered]
Trying to import some 2sxc apps into Oqtane.

CS0246: The type or namespace name 'HttpCookie' could not be found
There's no HttpCookie in System.Web?

**Comment by enfJoao** [comment]
Hey. This is still an issue for me...
I understand if this is a very basic question and probably has a really simple answer, but can anyone help me move past this issue?

**Comment by sbwalker** [comment, marked as answer]
@enfJoao System.Web is a .NET Framework artifact... whereas Oqtane is built on .NET Core. I think the best place to ask questions about 2sxc is on their website: https://2sxc.org/en/contact

**Comment by enfJoao** [comment]
Well... That explains it... I thought the hard part about migrating 2sxc apps would be adapting the framework (users, permissions, view urls, etc). I guess all the apps will need a lot of work to update. Is there a cheat sheet to framework => core?
Thank you for your time.
